src/annepro2.rs

```diff
@@ -66,8 +63,8 @@ pub fn flash_firmware<R: std::io::Read>(target: AP2Target, base: u32, file: &mut
             }
             let dev = api.device_list().find(|dev| {
                 dev.vendor_id() == ANNEPRO2_VID &&
-                    ((dev.usage() == USAGE_C15 && dev.usage_page() == USAGE_PAGE_C15 && dev.product_id() == PID_C15) ||
-                     (dev.usage() == USAGE_C18 && dev.usage_page() == USAGE_PAGE_C18 && dev.product_id() == PID_C18))
+                    ((dev.usage() == USAGE_C15 && dev.product_id() == PID_C15 && dev.interface_number() == 1) ||
```

Comment:
If we are not checking `usage_page` then we should remove the check for `usage` as well.